tools/format.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+set -eu
+
+
+cd "$(git rev-parse --show-toplevel)"
+
+echo -e "===> check flake8"
+flake8 onnx
```

Comment:
Shall we use `flake8` instead of `flake8 onnx` to cover more files under ONNX repo? For instance, the files [here](https://github.com/onnx/onnx/tree/main/workflow_scripts).

Reply:
Since just using `flake8` also affects files in CI environment, I changed this to `flake8 onnx tools workflow_scripts` (also, it will also prevent checking files in `.venv/` in your local environment).